src/quo/components/links/internal_link_card/community/view.cljs

```diff
@@ -0,0 +1,96 @@
+(ns quo.components.links.internal-link-card.community.view
+  (:require
+    [quo.components.community.community-stat.view :as community-stat]
+    [quo.components.links.internal-link-card.community.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description members-count active-members-count]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description} description]
```

Comment:
I literally have this style guide link in my favorites ;) It's a very common stylistic mistake in PRs.

> Vertically align function (macro) arguments spanning multiple lines.
>
> -- https://guide.clojure.style/#vertically-align-fn-args